fili-core/src/main/java/com/yahoo/bard/webservice/util/SimplifiedIntervalList.java

```diff
@@ -103,6 +104,14 @@ public static SimplifiedIntervalList simplifyIntervals(Collection<Interval>... i
                 .collect(getCollector());
     }
 
+    /**
+     * Static method to provide an empty list.
+     *
+     * @return a new, empty list.
+     */
+    public static SimplifiedIntervalList empty() {
```

Comment:
If we want this to be used, we should have the constructor be private. Otherwise, we should just remove this method and make everything use the constructor. No need for both, is there?